src/BloomBrowserUI/bookEdit/js/readerToolsModel.ts

```diff
@@ -1063,6 +1063,21 @@ class ReaderToolsModel {
       returnVal.push(new DataWord(words[i]));
     }
 
+    // inform the user if the list was truncated
+    var accordion: Window = iframeChannel.getAccordionWindow();
+    var msgDiv: JQuery = $(accordion).find('#allowed-word-list-truncated');
+
+    // We are reducing the max by 10 here in case there were a couple of duplicates that were removed after the list
+    // was truncated, causing the total number of words to now be slightly less than the max.
+    if (words.length < model.maxAllowedWords - 10) {
```

Comment:
This is confusing. I thought you said above that the server is now dealing with duplicates. So how can we end up removing more? Also, I suppose it's very unlikely, but if someone actually does have 9995 words, they will get a spurious message...I don't suppose that matters all that much, but if we don't need this at all I'd rather not have it.